hack/swagger/go.mod

```diff
@@ -4,8 +4,8 @@ go 1.20
 
 require (
 	github.com/kubeflow/training-operator v0.0.0-00010101000000-000000000000
-	k8s.io/klog/v2 v2.90.1
-	k8s.io/kube-openapi v0.0.0-20230501164219-8b0f38b5fd1f
+	k8s.io/klog/v2 v2.100.1
+	k8s.io/kube-openapi v0.0.0-20230717233707-2695361300d9
```

Comment:
I am wondering do we really need separate go.mod file for Swagger ?

Reply:
I actually didn't plan to touch this part as I wanted to keep the structure as is.